panoptes_aggregation/reducers/polygon_reducer.py

```diff
@@ -0,0 +1,205 @@
+'''
+Polygon/Freehand Tool Reducer Using DBSCAN
+------------------------------------------
+This module provides functions to reduce the polygon extractions from
+:mod:`panoptes_aggregation.extractors.polygon_extractor` using the
+algorithm DBSCAN.
+'''
+from sklearn.cluster import DBSCAN
+import numpy as np
+from collections import OrderedDict
+from .reducer_wrapper import reducer_wrapper
+from .polygon_reducer_utils import IoU_metric_polygon, cluster_average_last, \
+    cluster_average_intersection, cluster_average_union, \
+    cluster_average_median, IoU_distance_matrix_of_cluster, \
+    IoU_cluster_mean_distance
+from .text_utils import tokenize
+import warnings
+import shapely
+
+with warnings.catch_warnings():
+    # collatex is a bit old, we can safely ignore this message as the display
+    # functions are optional and never used in this code
+    warnings.filterwarnings('ignore', category=DeprecationWarning, message='Importing display')
+    import collatex as col
+
```

Comment:
This is only needed in the text transcription file, I guess it is a holder over from a copy-paste job.